### PR TITLE
QA Issues

### LINKED ISSUE #24: Separate Blue Background from Main Illustration
The blue background should be 100% width of the screen.
And the width of the illustration should be the same as the content width.

I created the assets in figma!
https://www.figma.com/file/OcNiuQUHT7CWXdOmxGDuXH/Seaborne-Sprint?node-id=569%3A1

### LINKED ISSUE #25: Black Font On All Headers
Turn all these headers black

Screen Shot 2020-09-28 at 2 04 48 PM
Screen Shot 2020-09-28 at 2 04 55 PM
Screen Shot 2020-09-28 at 2 04 37 PM
images.githubusercontent.com/58054507/94477826-65d84700-01a0-11eb-94df-982e1fbe9a63.png">
Screen Shot 2020-09-28 at 2 04 35 PM

### LINKED ISSUE #26: Desktop: Max-width 80 rem
Screen Shot 2020-09-28 at 3 58 08 PM

### LINKED ISSUE #27: Text Body Sm to 1 rem
Screen Shot 2020-09-28 at 4 04 19 PM

### LINKED ISSUE #28: Desktop: Headline size
Screen Shot 2020-09-28 at 4 08 07 PM

### LINKED ISSUE #29: Bold Weight from 700 to 500
Screen Shot 2020-09-28 at 4 15 17 PM

It would be "Be Vietnam Medium"

https://fonts.google.com/specimen/Be+Vietnam?sidebar.open=true&selection.family=Be+Vietnam:wght@500

### LINKED ISSUE #30: Desktop: Footer 1rem
Turn the font to 1 rem.

Screen Shot 2020-09-28 at 4 17 12 PM

### LINKED ISSUE #31: Footer border color
Screen Shot 2020-09-28 at 4 25 40 PM

### LINKED ISSUE #32: Desktop: Text Subheader Size
Screen Shot 2020-09-28 at 4 29 34 PM

### LINKED ISSUE #33: Tablet: Match font sizes
If we could just match the type sizes in the tablet version of the site to the designs. 

https://www.figma.com/file/OcNiuQUHT7CWXdOmxGDuXH/Seaborne-Sprint?node-id=292%3A0

### LINKED ISSUE #37: Button font weight/height consistency
Can we make the buttons font weight the "500" weight. 
And make the type size  and button height of the "copy to clipboard" button the same as the "Let's get in touch"

Screen Shot 2020-09-30 at 4 51 39 PM

### LINKED ISSUE #38: Copy to clipboard button say "copied"
When you click the copy to clipboard button can it turn blue and say "Copied!"

### PR DESCRIPTION
**Pull request by chrismekim**
- This PR is for the QA issues below, mostly font sizes, color changes, button styling adjustments, and makes the "Copied to clipboard" button on the contact page say "Copied!" when it is copied. 

Closes #24 
Closes #25
Closes #26
Closes #27
Closes #28
Closes #29
Closes #30
Closes #31
Closes #32
Closes #33
Closes #37
Closes #38

### CODE REVIEW

**Comment by brendonsanctuary** [comment]
Can you replace the two header assets with new assets

**Review by brendonsanctuary — COMMENTED** [review]
The only thing I am seeing since we swapped those images out is the design breaks on tablet and mobile now.

**Review by whoisgina — APPROVED** [review]
Code looks good and FeaturePeek too, except that in the offerings section `Assesment` should be `Assessment`.

**Review by brendonsanctuary — APPROVED** [review]

File: src/partials/head.html [hunk 1]
```diff
@@ -2,7 +2,11 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link href="css/main.scss" rel="stylesheet" />
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Be+Vietnam" />
+  <link href="https://fonts.googleapis.com/css?family=Be+Vietnam" rel="stylesheet" />
```
Comment (mokaymokay) [inline src/partials/head.html]: I'd add `&display=swap` here too if you wish to use a fallback before the font gets loaded

**Review by mokaymokay — APPROVED** [review]
LGTM after the typo fix that @whoisgina mentioned

**Comment by chrismekim** [comment]
@whoisgina @mokaymokay Thanks for the reviews! I fixed the typo and added display swap :)

**Pull request merged**